.github/workflows/scripts/ti_build/llvm.py

```diff
@@ -30,12 +29,8 @@ def setup_llvm() -> str:
 
     match (u.system, u.machine):
         case ("Linux", "x86_64"):
-            if cmake_args.get_effective("TI_WITH_AMDGPU"):
```

Comment:
we cant use this old v15 amd llvm if we are migrating to a newer version of llvm. so we have to remove it.

I suppose one question is, can we included amd in our llvm sdk builds 🤔